packages/fxa-auth-server/fxa-oauth-server/lib/keys.js

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const config = require('./config');
+const { jwk2pem } = require('pem-jwk');
+
+
+function pub(key) {
+  // Hey, this is important. Listen up.
+  //
+  // This function pulls out only the **PUBLIC** pieces of this key.
+  // For RSA, that's the `e` and `n` values.
+  //
+  // BE CAREFUL IF YOU REFACTOR THIS. Thanks.
+  return {
+    kty: key.kty,
+    alg: 'RS256',
+    kid: key.kid,
+    'fxa-createdAt': key['fxa-createdAt'],
+    use: 'sig',
+    n: key.n,
+    e: key.e,
+  };
+}
+
+const PRIVATE_KEYS_RAW = [config.get('openid.key')];
```

Comment:
nit: for uninitiated readers, it might be clearer to use `_JSON` rather than `_RAW` as a suffix on these; it would be easy to think that `_RAW` indicated e.g. a buffer rather than an object.

Reply:
Oh, or `_OBJ` like you use in some below.